starter-code/main.js

```diff
@@ -0,0 +1,147 @@
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) => {
+        switch(option){
+          case "1":
+						db.collection('companies').find({}, {name: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							}
+						});
+						break;
+          case "2":
+						db.collection('companies').find().toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							} else {
+								console.log(result.length);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							}
+						});
+						break;
+					case "3":
+						db.collection('companies').find({"founded_year": 2004}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							} else {
+								console.log(result.length);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							}
+						});
+						break;
+					case "4":
+						db.collection('companies').find({"founded_year": 2004, "founded_month": 2}, {name: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							}
+						});
+						break;
+					case "5":
+						db.collection('companies').find({
+																				$and: [
+																					{"founded_year": 2004},
+																					{"founded_month": {$gt: 3}},
+																					{"founded_month": {$lt: 7}}
+																				]
+																			}, {name: 1, _id: 0}).sort({"founded_month": 1, "founded_day": 1}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							}
+						});
+						break;
+					case "6":
+						db.collection('companies').find({"offices.city": "Barcelona"}, {name: 1, _id: 0}).toArray((error, result) => {
+							if (error) {
+								console.log(error);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							} else {
+								console.log(result);
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+							}
+						});
+						break;
+					case "7":
+						db.collection('companies').find({"offices.city": "Barcelona"}, {name: 1, _id: 0}).toArray((error, result) => {
```

Comment:
You are repeating the 6th case. For doing this, search about .sort() and .limit().